### ISSUE TITLE
Weird email formatting

### ISSUE THREAD

**Issue by Smedzlatko** [gessnerfl/fake-smtp-server#36]
Hi,

I am using the dockerized version of `fake-smtp-server` and I noticed that it has a weird text formatting when examining the email.

This is the RAW payload received by the fake smtp server:
```
Received: from EXAMPLE ([172.21.0.1])
        by fa452830e231
        with SMTP (SubEthaSMTP 3.1.7) id KJJXUSEC
        for admin@example.com;
        Tue, 05 Jan 2021 11:57:47 +0000 (GMT)
Date: Tue, 5 Jan 2021 12:57:47 +0100 (CET)
From: noreply@example.com
To: admin@example.com
Message-ID: <467546711.2.1609847867469@MMYSLIK>
Subject: Project Test finished with result WARNING
MIME-Version: 1.0
Content-Type: multipart/mixed; 
	boundary="----=_Part_0_1749930186.1609847867151"

------=_Part_0_1749930186.1609847867151
Content-Type: multipart/related; 
	boundary="----=_Part_1_1362894073.1609847867158"

------=_Part_1_1362894073.1609847867158
Content-Type: text/html;charset=UTF-8
Content-Transfer-Encoding: 7bit

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>Monitoring project results</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'/>
    <!-- use the font -->
    <style>
        body {
            font-family: 'Roboto', sans-serif;
            font-size: 14px;
        }
    </style>
</head>
<body>
    <p>
        <b>Project finished: </b>
        <a href="http://localhost:8020/">Test</a>
    </p>
    <p>
        <b>Processing finished ID: </b><span>2021-01-05T11:57:39.855551Z</span>
    </p>
    <p>
        <b>Result: </b><span>WARNING</span>
    </p>

    <table>
        <thead>
            <tr>
                <th>STRUCTURE</th>
                <th>ANOMALY</th>
                <th>DQ</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>
                    <span>SKIPPED</span>
                </td>
                <td>
                    <span>SKIPPED</span>
                </td>
                <td>
                    <span>WARNING</span>
                </td>
            </tr>
        </tbody>
    </table>

</body>
</html>
------=_Part_1_1362894073.1609847867158--

------=_Part_0_1749930186.1609847867151--
```

And the email was displayed like this:
![email](https://user-images.githubusercontent.com/6072828/103644227-4d718880-4f56-11eb-9f31-2c325b96cbbc.png)

There are some additional spaces and line breaks added for no reason. Do you have any idea what might be causing this?

**Comment by gessnerfl** [comment]
I was able to reproduce it. The issue is caused by mixed CSS provided by the web ui and the email

**Comment by gessnerfl** [comment]
Fixed with release 1.7.0. @Smedzlatko can you please cross check

**Comment by Smedzlatko** [comment]
Hi, thanks for the fix. I tested today in the dockerized version and it is working now.
